src/CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ if (“${PROJECT_SOURCE_DIR}” STREQUAL “${CMAKE_SOURCE_DIR}”)
     # Set some default BLT options before loading BLT only if not included in
     # another project
     if (NOT BLT_CXX_STD)
-        set(BLT_CXX_STD "c++11" CACHE STRING "")
+        set(BLT_CXX_STD "c++14" CACHE STRING "")
```

Comment:
Is this a strict requirement for inlet / sol ? Or a nice-to-have feature ?

I think this change should be done is a separate PR where it can be tested/validated separately.

Reply:
Also, not all of our users are on `C++14` yet, so we should confirm that this will not be a breaking update.

(I am not expecting problems, but we've been burned in the past, and do not want to unnecessarily introduce problems that will prevent our users from upgrading)

Reply:
According to their docs (which they were nice enough to build for us), it should work with C++11 so we can safely revert this change.  This happened early when we weren't sure. 

https://sol2.readthedocs.io/en/v2.20.6/compilation.html

Reply:
Though @gzagaris 's PR will soon require C++14 due to raja being updated.  At some point we are going to have to bite that bullet.

Reply:
C++ 14 was a requirement for sol

Reply:
That's correct,  C++14 is required (though no strongly enforced) with newer RAJA. Consequently, everyone is going to be using C++14. All our compilers already support it. We should run with this.

Reply:
Thanks @gzagaris.  Per our team meeting on 9/14, we will hold off on requiring `C++14` for axom for a few months, but can certainly optionally build with `C++14`